src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -35,6 +38,15 @@ internal static bool AcceptedTermsOfUse(DynamoViewModel dynViewModel)
             return termsOfServiceAccepted;
         }
 
+        internal static bool IsPackageInstalled(PackageManagerSearchViewModel viewModel = null)
+        {
+            if (viewModel == null)
+                return true;
+
+            bool canInstall = viewModel.CanInstallPackage(Res.AutodeskSamplePackage);
+            return !canInstall;
+        }
```

Comment:
Can we implement this as a static property with only a getter?

Reply:
We can't because the validation needs to be a function. The validation was done to differentiate the params of the methods. I didn't have another solution in mind for that case.